lib/crypto/packaged.py

```diff
@@ -153,7 +151,8 @@ def sign_multi(file_path, endpoint, guid):
             xpi_info = parse_xpi(xpi, check=False)
             if xpi_info['type'] == amo.ADDON_EXTENSION:
                 # Sign the internal extensions in place.
-                cert_serial_nums.append(call_signing(xpi, endpoint, guid))
+                cert_serial_nums.append(
+                    call_signing(xpi, endpoint, xpi_info['guid']))
```

Comment:
I'm not sure I understand the concern here: all the inner extensions will still be reviewed, right? So they're only signed if the review is accepted. What does the GUID (and owning it or not) has to do with it?

Another solution would be to check if the guid here (of the include extension) matches the guid of the outer (multi-package) extension, and only sign if it does. In that case, we'll need some modification of the amo-validator to warn/fail validation if the inner extensions that doesn't match aren't signed, to warn the dev that his multi-package won't be fully installable.